pinephone/vibrator/FFDevice.cpp

```diff
@@ -0,0 +1,135 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (C) 2019 The Android Open-Source Project
+ * Copyright (C) 2021 GloDroid project
+ */
+
+#include "vibrator-impl/FFDevice.h"
+
+#include <android-base/logging.h>
+
+typedef bool (*dirfunc_t)(const char *path, void *param);
```

Comment:
Prefer `using <alias-name> = <type>` over `typedef <type> <alias-name>` for C++ please.

Also, consider replacing `const char* path` with `std::string_view path` here, since this function is called in one place, but could be potentially used in many places, so it is better to do conversion at calling side, instead of callee side.

Reply:
Done,